### PR TITLE
updated all fulltext to full-text

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc [hunk 1]
```diff
@@ -113,7 +113,7 @@ this means that updates will be applied in a background thread "as soon as possi
 [NOTE]
 ====
 Using index-specific settings via the `config` parameter is experimental, because these settings currently do not get replicated in a clustered environment.
-See the <<operations-manual#index-configuration-fulltext-search, Fulltext Schema Indexes>> section of the Operations Manual, for how to configure the default fulltext index configurations in <<operations-manual#file-locations, _neo4j.conf_>>.
+See the <<operations-manual#index-configuration-fulltext-search, Full-text Schema Indexes>> section of the Operations Manual, for how to configure the default full-text index configurations in <<operations-manual#file-locations, _neo4j.conf_>>.
```
Comment (mariascharin) [inline cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc]: This is outside of this edit, but while we're making changes to this file I'd suggest changing this to something like:

See <<operations-manual#index-configuration-fulltext-search, Operations Manual -> Indexes to support fulltext search>> for instructions on how to configure full-text indexes in <<operations-manual#file-locations, _neo4j.conf_>>.

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc [hunk 2]
```diff
@@ -160,8 +160,8 @@ RETURN node.title, score
 2+d|4 rows
 |===
 
-The fulltext schema indexes uses Lucene under the hood.
-This means that we can use Lucene's fulltext query language to express what we wish to search for.
+The full-text schema indexes uses Lucene under the hood.
```
Comment (mariascharin) [inline cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc]: Similar to the previous comment - I suggest changing this to something similar to:

Full-text schema indexes are powered by the http://lucene.apache.org/[Apache Lucene] indexing and search library.

**Review by mariascharin — APPROVED** [review]
Thank you @AlexicaWright !
I have suggested two changes. I would have pushed them to your branch and merged it myself, but for some reason I am asked for a password to your account. Something odd going on there...

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @AlexicaWright and @mariascharin 

I think the problem is caused by Jessica not being in the `Neo4j Dev Team` list. I've requested that her profile is added.

If I can resolve today, I will push the changes suggested and merge.

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc (hunk 1 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc]: I agree! Thank you @mariascharin!

File: cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc (hunk 2 above)
Comment (AlexicaWright) [reply cypher/cypher-docs/src/docs/dev/ql/schema-index/fulltext-schema-index.adoc]: Ditto! Thank you @mariascharin!

**Pull request merged**